contract-examples/local_rpc.example.json

```diff
@@ -1,4 +0,0 @@
-{
-  "rpc": "http://127.0.0.1:9650/ext/bc/dRTfPJh4jEaRZoGkPc7xreeYbDGBrGWRV48WAYVyUgApsmzGo/rpc",
-  "chainId": 43214
-}
```

Comment:
why did we remove this? This is linked in the contract-examples README.

Reply:
This was replaced to switch to using environment variables, so that we do not need to overwrite this file in our tests, which seemed pretty jank to me.

This is now handled here: https://github.com/ava-labs/subnet-evm/blob/refactor-e2e-tests/contract-examples/hardhat.config.ts#L6-L7.

Reply:
Ok, sweet! We should probably update that specific README

Reply:
Still need to cleanup the README. Good callout that I'll need to update the contract-examples README as well.

Reply:
Updated the READMEs, thanks for flagging this 🙌